vendor/assets/stylesheets/dvl/components/sidebar_data.scss

```diff
@@ -61,10 +61,16 @@
 
 .sidebar_data_text {
   margin-left: $lineHeight;
-  line-height: $lineHeight;
```

Comment:
I don't understand? The absolute line-height here is what was causing the issue.